CompilerSource/ipc/Path.fbs

```diff
@@ -0,0 +1,23 @@
+
+struct PathPoint {
+  x:int;
+  y:int;
+  speed:int;
+}
+
+table Path {
+  name:string;
+  id:int;
+
+  smooth:bool;
+  closed:bool;
+  precision:int;
+
+  //backgroundRoomId:int; //This was used by LGM rather than ENIGMA; it doesn't need to be here.
+  snapX:int; //TODO: Find out if these are needed and decide on keeping them
+  snapY:int;
+
+  points:[PathPoint];

```

Comment:
This here is going to be a vector basically, is that ok? I can't find anywhere that FlatBuffers has a plain-old array field type. Because of this, we don't need to write the pointCount, should we anyway? Also consider that Image will have a vector of bytes. Is there any performance overhead involved here from not using plain arrays?